### ISSUE TITLE
Icons within timeline content are incorrectly sized

### ISSUE THREAD

**Issue by bcallaghan-et** [vmware/clarity#5366]
When placing an icon within the title or description of a timeline step, the icon inherits the same size as the step's main icon. This is unintuitive and makes styling icons within the content difficult if not impossible. I can reproduce this behavior in a [Stackblitz](https://stackblitz.com/edit/clarity-v4-timeline-icons?file=src/app/app.component.html). All four icons in this sample are supposed to be different sizes, yet they all use the size inherited from the timeline.

Angular 10.2.3
Clarity 4.0.7
Chrome 87

![image](https://user-images.githubusercontent.com/44448874/101225082-c88d0980-364d-11eb-9a70-249a678fae5e.png)

**Comment by bdryanovski** [comment]
@bcallaghan-et  thanks for reporting this, we gonna need some time to investigate the issue I manage to reproduce it so it's a valid issue, keep in mind that in December we will be with limited staff for the holidays, so it could take a little bit more.

**Comment by hippee-lee** [comment]
Hi Ben, it looks like this is an issue with min width/height set for timeline main step icon. I forget why I added that so as Bozhidar mentioned we need to investigate to see if we can make the css more specific. 

Until then, I came up with a workaround [here](https://stackblitz.com/edit/gh-5366-icon-size-in-timeline?file=src/app/app.component.html). What it does is unset the min height/width for the projected icons in the step description but that could be done for any of they child components. This allows the icon to be made smaller. 

If the need is only to make them bigger, that already works, just add `size="64"` to any projected `clr-icon`. 

Will this work for you?

**Comment by bcallaghan-et** [comment]
@hippee-lee That workaround mostly works for me. I added the following CSS to the root of my project, and it seems to work well. I had to re-add the height and width of alert-icons to restore their normal look, and there's probably other components that are similarly affected.
```scss
.clr-timeline-step * clr-icon {
  // Remove the height and width of icons that are indirectly in a timeline.
  height: unset;
  width: unset;
  min-height: unset;
  min-width: unset;

  &.alert-icon {
    // Manually fix alert-icons to their normal size.
    height: $clr_baselineRem_1;
    width: $clr_baselineRem_1;
  }
}
```

Looking over the [source](https://github.com/vmware/clarity/blob/next/packages/angular/projects/clr-angular/src/timeline/_timeline.clarity.scss#L20-L58) that generates the timeline icon size, it appears to me that these styles were only intended to affect the one icon that is a direct part of the timeline. Would it work if that assumption was baked into the styles? For example, the current selector `cds-icon` could be replaced with `> cds-icon` so it only affects direct children and not any icons buried within the step title or step description.

**Comment by hippee-lee** [comment]
I believe they were intended to work with the timeline icons but this component came from an external contributor so I'm not 100% sure. Yeah, the fix is likely to increase the specificity of the css. 

Thanks for finding and reporting this.

**Comment by gnomeontherun** [comment]
This should have been addressed by #6139 closing as resolved.
